Utilities/bin_patch.cpp

```diff
@@ -1667,9 +1695,7 @@ void patch_engine::save_config(const patch_map& patches_map)
 
 	out << YAML::EndMap;
 
-	fs::pending_file file(path);
```

Comment:
why are you making the write non-atomic?

Reply:
Why should it be atomic? Also, why isn't the write_file method not doing that if it's important ?

Reply:
Well, I don't have the patience to refactor it right now, so I'll revert it